README.md

```diff
@@ -1 +1,26 @@
-# hw01
\ No newline at end of file
+# hw01
+## About Me
+HELLO! My name is **Jarret Angbazo** and I am a second-year MPP student at the *Harris School of Public Policy*. I was born in Nigeria and before moving to Chicago for graduate school, I grew up in Maryland. Chicago is the furthest west on the continental United States that I've lived.
+
+Quick two truths and a lie:
+
+1. I did this assignment in 5 minutes
+
+2. I don't hate the player, I hate the game
+
+3. I ran a personal best in the 5k this weekend
+
```

Comment:
You don't need to number the list elements. Markdown will automatically number them.

```
1. I did this assignment in 5 minutes
1. I don't hate the player, I hate the game
1. I ran a personal best in the 5k this weekend
```

1. I did this assignment in 5 minutes
1. I don't hate the player, I hate the game
1. I ran a personal best in the 5k this weekend